aviary/utils/Fortran_to_Aviary.py

```diff
@@ -434,6 +434,13 @@ def update_gasp_options(vehicle_data):
         input_values.set_val(Aircraft.Wing.FLAP_DRAG_INCREMENT_OPTIMUM,
                              [[.12, .23, .13, .23, .23, .1, .15][flap_ind]])
 
+    res = input_values.get_val(Aircraft.Design.FIXED_RESERVES_FUEL, units='lbm')[0]
+    if res <= 1.0:
+        input_values.set_val(Aircraft.Design.FIXED_RESERVES_FUEL, [0], units='lbm')
+        input_values.set_val(Aircraft.Design.RESERVES_FRACTION, [res], units='unitless')
+    else:
+        input_values.set_val(Aircraft.Design.RESERVES_FRACTION, [0], units='unitless')
+
```

Comment:
```suggestion
    if res <= 0:
        input_values.set_val(Aircraft.Design.FIXED_RESERVES_FUEL, [0], units='lbm')
        input_values.set_val(Aircraft.Design.RESERVES_FRACTION, [-res], units='unitless')
    elif res > 10:
        input_values.set_val(Aircraft.Design.RESERVES_FRACTION, [0], units='unitless')
    else:
    	ValueError('"FRESF" is not valid between 0 and 10.')
```

Reply:
We have decided not to allow res to go negative.

Reply:
In Fortran to Aviary, a negative value for FRESF still needs to be supported, but this stores RESERVES_FRACTION as a positive number